### DISCUSSION TITLE
Activate a GPIO if MaxRateValue is exeeded

### DISCUSSION THREAD

**Discussion by PeterMau19** [jomjol/AI-on-the-edge-device#2888] [Ideas]
Idea:
if MaxRateValue or similar value is exeeded an no other errors occured, enable to change value of a free GPIO (12) to activate a motorized valve in the main pipe or give a alarm to some device.

**Comment by caco3** [comment]
I would suggest you use a separate device and a home automation system for this. I can suggest Homeassistant which is supported by our device out of the box.
